s4e-web/src/app/views/apihowto/apihowto.component.html

```diff
@@ -49,7 +51,7 @@ <h2>Pobranie klucza API (tzw. token)</h2>
 
   <label>Przykład:</label>
   <code>
-    curl -LX POST '{{ API_BASE }}/token' -H 'Content-Type: application/json'
+    curl -LX POST '{{ API_BASE }}/token' --header 'Content-Type: application/json'
```

Comment:
is there any difference between -H and --header? curl help doesn't mention anything :(

Reply:
It's just more consistent, both in wget and curl you have --header, but only in curl there is -H which expands to --header (it's something completely different in wget).